### ISSUE TITLE
Conversion from AZW3 to PDF/ePUB failed.

### ISSUE THREAD

**Issue by temp3709** [apprenticeharper/DeDRM_tools#817]
I am currently having a problem while trying to convert a .azw3 file to .pdf or .ePUB. None of them work. I have used Kindle v17.44170 to download the ebook and Calibre v3.42 with DeDRM v6.6.3. Judging from the logs, the drm removal seems to be working fine unlike the conversion. I am also able to view the ebook in Calibre after importing it. Any idea on whats causing a problem when converting?

Logs:
[Import log.txt](https://github.com/apprenticeharper/DeDRM_tools/files/3132503/Import.log.txt)
[Conversion log.txt](https://github.com/apprenticeharper/DeDRM_tools/files/3132502/Conversion.log.txt)

**Comment by j-howell** [comment]
The import and DRM removal were successful.

> MemoryError

The 64-bit version of calibre is needed to convert large and complex books. You have the 32-bit version installed.

**Comment by temp3709** [comment]
Thanks for the quick reply. I have installed the 64-bit version and it worked well past the point it used to crash. It ran around 20min until windows "flickered" and closed it, probably because it was exceeding available RAM. I closed all other programs and tried again until about an hour later the same happened, but this time it continued after the memory error. A few minutes later, i closed the log and it stopped the conversion, it may have been coincidence, i can't really tell. I have 8GB RAM and am hoping it should be enough, even though it is a rather large file with 142MB.

I have read somewhere else that the best practice should be to just let it run until it finishes, no matter how long it takes, but i am kinda scared that it will lead to a memory error after hours and lead to wasted time. Any suggestions?
